packages/client/src/Call.ts

```diff
@@ -237,7 +238,24 @@ export class Call {
           debounce((v) => timer(v.type)),
           map((v) => v.data),
         ),
-        (subscriptions) => this.sfuClient?.updateSubscriptions(subscriptions),
+        (subscriptions) => {
+          if (!this.sfuClient) return;
+
+          runWithRetry(this.sfuClient.updateSubscriptions, {
+            retryAttempts: 30,
+            delayBetweenRetries: retryInterval,
+            didValueChange: (initialSubscriptions) => {
+              return (
+                initialSubscriptions !==
+                this.trackSubscriptionsSubject.getValue().data
+              );
+            },
+            isRetryable: (error) => {
+              // decide whether to retry execution
+              return true;
+            },
```

Comment:
@oliverlaz We should probably adjust this part and generate some reusable `isRetryable` functions.

Reply:
We can have one that inspects SFU's `response.error.shouldRetry` as such instruction can be returned by the SFU even when the RPC succeeded (HTTP 200). Check `StreamSfuClient#retryable`.